train_configs/debug_model.toml

```diff
@@ -9,6 +9,7 @@ use_for_integration_test = true
 enable_profiling = true
 save_traces_folder = "profile_trace"
 profile_freq = 10
+enable_memory_snapshot = false
```

Comment:
existing `.toml` without `enable_memory_snapshot` still works. since `enable_memory_snapshot` is optional with `getattr(config.profiling, 'enable_memory_snapshot', False)` I am just adding it here so people can start toggle it

Reply:
ditto: we should put default option `False` into config_manager, and remove this option in all the toml config files. Maybe only enable it to `True` in debug_model.